Objects/EventItem/EventItemSpawner.gd

```diff
@@ -0,0 +1,27 @@
+extends Node
+
+var time_to_spawn_item: float = 5
+var temp_time: float = 0
+var SackOfGold = load("res://Objects/EventItem/SackOfGold.tscn")
+var DwarfInBalloon = load("res://Objects/EventItem/DwarfInBalloon.tscn")
+
+func set_next_item_time():
+	var value = randi()%60+20
+	time_to_spawn_item = value 
+	temp_time = 0
+
+func spawn_item():
```

Comment:
I think, there is better way to do that random spawning:

We keep all event items in one array in that file:
```gdscript
var event_items : Array = [
	load("res://Objects/EventItem/SackOfGold.tscn"),
	load("res://Objects/EventItem/DwarfInBalloon.tscn")
]
```
and in 'spawn_item' function just get random element from the array:
```gdscript
var index : int = randi() % event_items.size()
var item = event_items[index]
get_parent().call_deferred("add_child", item)
```

It's much more flexible. To add a new item we just passing next 'load' function in the array declaration